base/docs/helpdb/Base.jl

```diff
@@ -2350,6 +2450,33 @@ countlines
     .\\(x, y)
 
 Element-wise left division operator.
+
+```jldoctest
+julia> A = [1 0; 0 -1];
```

Comment:
Seems like an advanced/complicated example. Most often, this is probably used for scalar elements.